spec/unit/typesafe/config/path_spec.rb

```diff
@@ -0,0 +1,259 @@
+require 'spec_helper'
+require 'hocon'
+require 'test_utils'
+
+
+describe Hocon::Impl::Path do
```

Comment:
I don't think we really have a good way of indicating that beyond naming the file. We've been naming files similarly to what's in the Java library with, so for example, `PathTest.scala` becomes `path_spec.rb`.